lib/segment/analytics/defaults.rb

```diff
@@ -12,8 +12,8 @@ module Request
       end
 
       module Queue
-        BATCH_SIZE = 100
-        MAX_SIZE = 10000
+        BATCH_SIZE = 1000
```

Comment:
For our use case (sending large bulks of data via the API and no strict requirements on latency) I think it makes sense to send data in larger batches and to use a larger queue size. It would be probably better to also expose these two options in the client.